packages/orchestrator-ui-components/src/utils/date.ts

```diff
@@ -35,23 +34,44 @@ export const formatDate = (dateString: string | undefined) => {
     if (!dateString) return '';
 
     const date = parseDate(dateString);
-    return parseDateToLocaleString(date);
+    return parseDateToLocaleDateTimeString(date);
 };
 
-export const parseDateToLocaleString = (value: Date | null) =>
-    // Todo #95: Fix default locale (guess it via browser lang?)
-    value?.toLocaleString(DUTCH_LOCALE) ?? '';
+export const parseIsoString =
+    (dateToStringParser: (date: Date | null) => string) =>
+    (date: string | null) =>
+        dateToStringParser(parseDate(date));
 
-export const parseTimeToLocaleString = (value: Date | null) =>
-    // Todo #95: Fix default locale (guess it via browser lang?)
-    value?.toLocaleTimeString(DUTCH_LOCALE) ?? '';
+export const parseDateToLocaleDateTimeString = (value: Date | null) =>
+    value?.toLocaleString(getCurrentBrowserLocale()) ?? '';
 
-export const parseDateRelativeToToday = (dateString: string | undefined) => {
-    if (!dateString) return '';
-    const date = parseDate(dateString);
-    if (date?.toLocaleDateString() === new Date().toLocaleDateString()) {
-        return parseTimeToLocaleString(date);
-    } else {
-        return parseDateToLocaleString(date);
+export const parseDateToLocaleDateString = (value: Date | null) =>
+    value?.toLocaleDateString(getCurrentBrowserLocale()) ?? '';
+
+export const parseDateToLocaleTimeString = (value: Date | null) =>
+    value?.toLocaleTimeString(getCurrentBrowserLocale()) ?? '';
+
+export const isToday = (date: Date) =>
+    date.toLocaleDateString() === new Date().toLocaleDateString();
+
+export const parseDateRelativeToToday = (
+    date: Date | null,
+    shortNotation: boolean = false,
+) => {
+    if (!date) {
+        return '';
     }
+
+    if (isToday(date)) {
+        return parseDateToLocaleTimeString(date);
+    }
+
+    return shortNotation
+        ? parseDateToLocaleDateString(date)
+        : parseDateToLocaleDateTimeString(date);
 };
+
+export const parseDateStringRelativeToToday = (
+    dateString: string | undefined,
+    eitherDateOrTime: boolean = false,
```

Comment:
this looks funny, `dateOnly`?
It's difficult to read with the boolean here.

It might also be good to incorporate some of the ISO naming in the functions. As a lot of them map into a ISO string representation of a date. And you don't see that when reading the functions.

Reply:
I agree on the boolean. Why not just have two functions (with descriptive names) instead of carrying around a boolean parameter (with a default).

Reply:
Will split this up in 2 functions, that is indeed better

Reply:
done